dutch/bos/ontvoering/meehelpen/plassen/lopen/lopen.md

```diff
@@ -4,5 +4,5 @@ Wat doe je?
 Je laat je op de grond vallen en doet alsof je dood bent.
 
 Je klimt in een boom.
-
-Je stopt, [draait je om](omdraaien/terugrennen.md) en rent zo hard als je kan op de zware jongens af.
\ No newline at end of file
+Maar komt [vast te zitten](vastzitten/terugrennen.md).
+Je stopt, [draait je om](omdraaien/terugrennen.md) en rent zo hard als je kan op de zware jongens af.
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.